fixtures/dom/src/components/TestCase.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const propTypes = {
+  children: React.PropTypes.node.isRequired,
+  title: React.PropTypes.node.isRequired,
+};
+
+class TestCase extends React.Component {
+
+  render() {
+    const { title, description, children } = this.props;
+
+    return (
+      <section className="test-case">
+        <h2 className="type-subheading">
+          <label>
+            <input type='checkbox' />
+            {' '} {title}
+          </label>
+        </h2>
+        <p>{description}</p>
+
+        {children}
+      </section>
+    );
+  }
+}
+
+TestCase.propTypes = propTypes;
+
+TestCase.Steps = ({ children }) => (
+  <div>
+    <h3>Steps to reproduce:</h3>
+    <ol>
+      {children}
+    </ol>
+  </div>
```

Comment:
I think we're fine with only offering fixture tests for 0.13+, so we should probably convert all fixture components to use ES6 classes.